packages/rari/src/runtime/entry-client.ts

```
@@ -539,16 +551,12 @@ export async function renderApp(): Promise<void> {
 
     wrappedContent = React.createElement(
       ClientRouter,
-      // @ts-ignore - children passed as third argument; type checking varies based on build state
-      { initialRoute: window.location.pathname },
-      wrappedContent,
+      { initialRoute: window.location.pathname, children: wrappedContent },
```

Biome (2.4.6)

[error] Avoid passing children using a prop

(lint/correctness/noChildrenProp)